packages/api-plugin-promotions-discounts/src/discountTypes/item/applyItemDiscountToCart.js

```diff
@@ -85,6 +85,7 @@ export default async function applyItemDiscountToCart(context, params, cart) {
   }
 
   const affected = discountedItems.length > 0;
+  const reason = !affected ? "No items were discounted" : undefined;
```

Comment:
I'm a little confused by this, does this mean this will add a message every time a cart can't be discounted via offers or is this just for coupons or ???